src/main/frontend/app/containers/BarIndexContainer.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+
+import BarTile from '../components/BarTile'
+
+class BarIndexContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bars: []
+    }
+  }
+
+  componentDidMount() {
+    fetch('/api/v1/bars')
+    .then(response => {
+      return response.json()
+    })
+    .then(bars => {
+      this.setState( {bars: bars} )
+    })
+  }
+
+  // render() {
```

Comment:
let's remove this block of code as opposed to leaving the commented version in for SCM